### DISCUSSION TITLE
hardware support request

### DISCUSSION THREAD

**Discussion by slipttees** [thesofproject/sof#8848] [General]
I'm wondering about the possibility of implementing support for audio hardware that doesn't work on any Linux distribution. Therefore, you are forced to use Windows. I'm sorry for posting here if it's in the wrong place!

[Hardware Info](https://linux-hardware.org/?id=pci:8086-2284-10ec-11e2)

**Comment by plbossart** [comment]
Your system seems to be an Intel Braswell SoC with an HDaudio codec. This should work out of the box with the snd-hda-intel driver.

please attach the result of 'also-info' for us to see what happens.

**Comment by slipttees** [comment]
Thank you for your attention. I checked and the problem was hardware, I replaced it and it is working. My bad, thanks
